content/00.front-matter.md

```diff
@@ -34,7 +34,7 @@ on {{date}}.
   {%- endif %}<br>
   <small>
   {%- if author.affiliations is defined %}
-     {{author.affiliations}}
+     {{author.affiliations | join('; ')}}
```

Comment:
This looks good for the initial merge.  We can make a shorter version per https://github.com/greenelab/deep-review/issues/700#issuecomment-353462637 in a separate update.